cucumber/api/features/step_definitions/authz_steps.rb

```diff
@@ -82,7 +82,7 @@
 end
 
 Given(/^I create a binary secret value?$/) do
-  @value = Random.new.bytes(16)
+  @value = Random.new.bytes(16).force_encoding(Encoding::UTF_8)
```

Comment:
Yes, binary is `Encoding::ASCII_8BIT`, which is exactly what you get from `Random#bytes`. That's also what the service should return.